src/HatchIntakeSubsystem.cpp

```diff
@@ -31,44 +31,24 @@ void HatchIntakeSubsystem::teleop() {
     // Actuates the intake first then changes the diection of the intake wheel based 
     // on which button is pressed, the motor turns off when the hatch is either in 
     // or out depending on the button pressed.
-    // Right Button is spit hatch out and Right Trigger is take hatch in.
+    // Right Button is to outtake the hatch and Right Trigger is take hatch in.
     if (operatorJoystick->GetButton(CORE::COREJoystick::JoystickButton::RIGHT_BUTTON)) {
-        SetHatchActuator(false);
-        SetIntake(false, HatchIn());
+        SetIntake(false, HatchIn(), false);
     }
     if (operatorJoystick->GetButton(CORE::COREJoystick::JoystickButton::RIGHT_TRIGGER)) {
-        SetHatchActuator(false);
-        SetIntake(true, !HatchIn());
+        SetIntake(true, !HatchIn(), false);
```

Comment:
Is the hatch intake parameter supposed to be using the '!' symbol? Right now, this would mean that the value that is passed in would be the opposite of the current state of the hatch.